packages/client/src/views/Home/RecordAudit.tsx

```diff
@@ -240,6 +240,48 @@ const getLocation = (
   return ''
 }
 
+/*const getSavedApplications = (props: IFullProps): IApplicationData => {
+  const savedApplications = props.savedApplications
+  const applicationId = props.match.params.applicationId
+
+  const applications = savedApplications
```

Comment:
Do you want to delete this commented code?

Reply:
@atiqzaman-dsi vai, fixed them by merging the branch with develop.